application/jupyter/jupyter.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: jupyter
+  labels:
+    app: jupyter
+spec:
+  ports:
+  - port: 80
+    name: http
+    targetPort: 8888
+  selector:
+    app: jupyter
+  type: LoadBalancer
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: jupyter
+  labels:
+    app: jupyter
+spec:
+  containers:
+    - name: jupyter
+      image: skippbox/jupyter:${jupyter_version}
```

Comment:
Use the official [docker image](https://hub.docker.com/r/jupyter/base-notebook) from Jupyter.